scripting/ServerAdvertisements.sp

```diff
@@ -207,6 +209,19 @@ public int hSALangMenu(Menu menu, MenuAction action, int client, int Position)
 
 	return 0;
 }
+
+// Cookies Handler Menu
+public void CookieHandler(int client, CookieMenuAction action, any info, char[] buffer, int maxlen)
+{
+	switch (action)
+	{
+		case CookieMenuAction_SelectOption:
+		{
+			Command_ChangeLanguage(client, 0);
```

Comment:
could you create a function like ShowSetttingsMenu and put the menu part inside it because its confusing, i thought u tried to call a command inside a menu:
- https://github.com/srcdslab/sm-plugin-ServerAdvertisements/blob/1674b4f6af0e1038c66103ebac0844dff373f1b8/scripting/ServerAdvertisements.sp#L134
- https://github.com/srcdslab/sm-plugin-TopInfectors/blob/923b0ef70a634b0f3dfe53e8c3fac4cc0e4d3eda/addons/sourcemod/scripting/TopInfectors.sp#L438

Reply:
Yes that the plan, but the code need a refactor in the same time, it's messy.